flask_resty/spec/plugin.py

```diff
@@ -0,0 +1,45 @@
+from collections import defaultdict
+
+from flask import current_app
+from flask.ext.resty.spec.operation import Operation
+from .swagger_utils import flaskpath2swagger
+
+RESTY_PLUGIN_NAME = 'resty'
```

Comment:
Pull this out to something like https://github.com/mitsuhiko/flask-sqlalchemy/blob/master/flask_sqlalchemy/__init__.py#L608-L613 (maybe in a separate top-level `.utils`).